RxSwift/Observables/WithLatestFrom.swift

```diff
@@ -9,13 +9,13 @@
 extension ObservableType {
 
     /**
-     Merges two observable sequences into one observable sequence by combining each element from self with the latest element from the second source, if any.
```

Comment:
can you please revert the `, if any` portions throughout?
Aside for that, looks good. Thank you !

Reply:
@freak4pc  Could you please explain what the `if any` portion is supposed to communicate?

`Merges two observable sequences into one observable sequence by combining each element from self with the latest element from the second source, if any.`

I interpret this to mean that the latest element from the second source is optional.
In other words: If the second source has no latest element, the result observable would still fire an event.
However this is not true. If the second source has no latest element, the result observable will not fire an event.

Reply:
If you want to be more verbose you should say `[...] from the second source. if the second source has no latest element, no combined value will be emitted by this operator`.